<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DmxRangeWidget</class>
 <widget class="QDialog" name="DmxRangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMX range selection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,1">
   <item row="0" column="1">
    <widget class="QSpinBox" name="universeSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="firstLabel">
     <property name="text">
      <string>First DMX channel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="universeLabel">
     <property name="text">
      <string>Universe</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="firstDmxChanSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="lastDmxChanSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lastLabel">
     <property name="text">
      <string>Last DMX channel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="channelsLabel">
     <property name="text">
      <string>Channel count</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="channelsSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DmxRangeWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DmxRangeWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
